### PR TITLE
Fixed incorrect use of single quotes in the install OpenSearch guide.

### PR DESCRIPTION
**Pull request by marknyny**
### Description
Fixed incorrect use of single quotes. 
Original documented example does not work correctly in Windows environment:
```
C:> curl https://localhost:9200/ -ku 'admin:MyStrongPassword123!'
Unauthorized

C:> curl https://localhost:9200/ -ku admin:MyStrongPassword123!
{
"name" : "82cc606ccdd6",
"cluster_name" : "docker-cluster",
"cluster_uuid" : "O6kAWlxqSpWu1zbCqfnBrg",
"
```
### Issues Resolved
Removed incorrect single quote in the document sample
 
### Version
2.16.0

### Frontend features
_If you're submitting documentation for an OpenSearch Dashboards feature, add a video that shows how a user will interact with the UI step by step. A voiceover is optional._ 

### Checklist
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by marknyny** [comment]
Minor error in curl command sample (Window cmd): No single quote needed for Windows curl command; double quote can be used instead.

Evidences:

C:\> curl https://localhost:9200 -ku 'admin:MyStrongPassword123!'
Unauthorized
 

C:\> curl https://localhost:9200 -ku admin:MyStrongPassword123!
{
  "name" : "82cc606ccdd6",
  "cluster_name" : "docker-cluster",
  "cluster_uuid" : "O6kAWlxqSpWu1zbCqfnBrg",

**Comment by kolchfa-aws** [comment]
Thank you, @marknyny! 

@peterzhuamazon Could you weigh in on this? We normally use bash when providing command-line examples. Is there a unified way to write this command that will work on all platforms or do we have to add a differently formatted command for Windows specifically? Thanks!

**Comment by peterzhuamazon** [comment]
> Thank you, @marknyny!
> 
> @peterzhuamazon Could you weigh in on this? We normally use bash when providing command-line examples. Is there a unified way to write this command that will work on all platforms or do we have to add a differently formatted command for Windows specifically? Thanks!

After observation and test on a windows server, I confirm user is likely testing on command prompt / CMD.exe.
On both mingw and pwsh terminal, single quote works fine to pass a literal string.

In CMD, afaik, single quote does not actually able to group values.
Instead, it will push the entire string as literal `'admin:MyStrongPassword123!'`
As you can see in this payload by curl:
```
C:\Users\Administrator>curl -vvv https://localhost:9200/ -ku 'admin:myStrongPassword123!'
*   Trying [::1]:9200...
* Connected to localhost (::1) port 9200
* schannel: disabled automatic use of client certificate
* ALPN: curl offers http/1.1
* ALPN: server did not agree on a protocol. Uses default.
* using HTTP/1.x
* Server auth using Basic with user ''admin'
```

vs

```
C:\Users\Administrator>curl -vvv https://localhost:9200/ -ku "admin:myStrongPassword123!"
*   Trying [::1]:9200...
* Connected to localhost (::1) port 9200
* schannel: disabled automatic use of client certificate
* ALPN: curl offers http/1.1
* ALPN: server did not agree on a protocol. Uses default.
* using HTTP/1.x
* Server auth using Basic with user 'admin'
```

Note that it is `''admin'` vs `'admin'` here, and it is a specific issues with CMD on Windows.

Suggestions would be to use either powershell/bash or just change to double quotes if you may.
Tho without any quote it will be interpret as a simple string, I normally would like to at least have double quote, to avoid issues like if there is spaces in the string which can cause argument getting split, or variable subtitutions, and potential issues with special characters.

Thanks.

File: _install-and-configure/install-opensearch/docker.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Before continuing, you should verify that Docker is working correctly by deployi
     ```
 1. Send a request to port 9200. The default username and password are `admin`.
     ```bash
-    curl https://localhost:9200 -ku 'admin:<custom-admin-password>'
+    curl https://localhost:9200 -ku admin:<custom-admin-password>
```
Comment (kolchfa-aws) [inline _install-and-configure/install-opensearch/docker.md:109]: @marknyny, per @peterzhuamazon's suggestion, can you change to double quotation marks?
```suggestion
    curl https://localhost:9200 -ku "admin:<custom-admin-password>"
```

**Comment by dblock** [comment]
Note that we have unquoted examples everywhere else. Does it make sense to quote just here?

https://github.com/search?q=repo%3Aopensearch-project%2Fdocumentation-website%20-ku&type=code

**Comment by kolchfa-aws** [comment]
We need to do this in all instances. @marknyny Do you want to take it on as part of this PR? Or I can put up a separate PR.

**Comment by marknyny** [comment]
I would suggest to have minimum necessary changes to make examples work correctly. This changes was to eliminate confusion and a missleading info. I am not sure there is a practical benifit of adding double quotes accross the whole document.

**Comment by kolchfa-aws** [comment]
I think @dblock is referring to this exact curl request in other documentation pages where the login/password are not quoted. I can take this on once we merge this PR.

**Comment by dblock** [comment]
> I think @dblock is referring to this exact curl request in other documentation pages where the login/password are not quoted. I can take this on once we merge this PR.

I think quoting of username:password depends on what the username and password are. If you have a % on Windows you'll want quotes. If you have a `"`, you'll want single quotes, etc. I think the more general case is not to include any quotes and let the user figure the special cases out.

**Review by kolchfa-aws — APPROVED** [review]
Thank you, @marknyny! Merging now.

**Pull request merged**